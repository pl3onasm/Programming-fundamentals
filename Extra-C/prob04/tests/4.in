1000 2050