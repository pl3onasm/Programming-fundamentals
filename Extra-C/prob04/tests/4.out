1057: 10000100001
1137: 10001110001
1193: 10010101001
1273: 10011111001
1317: 10100100101
1397: 10101110101
1453: 10110101101
1533: 10111111101
1571: 11000100011
1651: 11001110011
1707: 11010101011
1787: 11011111011
1831: 11100100111
1911: 11101110111
1967: 11110101111
2047: 11111111111
